hack/bazel-server.sh

```diff
@@ -1,5 +1,13 @@
-BAZEL_PID=$(bazel info | grep server_pid | cut -d " " -f 2)
-while kill -0 $BAZEL_PID 2>/dev/null; do sleep 1; done
-# Might not be necessary, just to be sure that exec shutdowns always succeed
-# and are not killed by docker.
-sleep 1
+ARCH=$(uname -m)
+
+# s390x builder image is not shipped with bazel.
+if [ $ARCH != "s390x" ]; then
```

Comment:
@vamsikrishna-siddu Looking at this part again, i think we need to change this to using an injected environment variable or a cli argument instead, since otherwise this script will not run on s390x when we are triggering e2e tests.
Since we are planning on using a non-public image with bazel included in the future.
Alternatively we could also make the condition check for if bazel binary is present, though that might be instable, for example when using bazelisk...

Reply:
Hi @jschintag  sure will do that

Reply:
Hi @jschintag  I am using an environment variable called KUBEVIRT_RUN_UNIT_TEST_NO_BAZEL and injecting into the docker container. if this variable KUBEVIRT_RUN_UNIT_TEST_NO_BAZEL is set to true. Then it will sleep for infinite time otherwise it will invoke the bazel in the bazel-server.sh. Let me know your thoughts on this?

Reply:
I looked at the code again and there is already an existing env variable given to the `hack/dockerized` script called `KUBEVIRT_NO_BAZEL`. It is also already set to true when invoking `make go-test`. See https://github.com/kubevirt/kubevirt/blob/main/Makefile#L79-L80.
I would like to use the existing variable instead of introducing a new, duplicated one.

Reply:
Hi @jschintag  I have done the changes to use KUBEVIRT_NO_BAZEL. Please review it again Thanks

Reply:
Thank you, looks good